src/Google/Service/Resource.php

```diff
@@ -193,7 +197,12 @@ public function call($name, $arguments, $expected_class = null)
         null,
         $postBody
     );
-    $httpRequest->setBaseComponent($this->client->getBasePath());
+
+    if ($this->rootUrl) {
```

Comment:
TBH, I could go either way. Considering that it's a completely undocumented parameter in public docs, and the only documented reference is in config.php:

      // Don't change these unless you're working against a special development
      // or testing environment.
      'base_path' => 'https://www.googleapis.com',

I could see it going either way. It's up to you.

I'll bump the Client.php version number in the PR as well.